website/docusaurus.config.js

```diff
@@ -24,6 +24,9 @@ const lightCodeTheme = require('prism-react-renderer/themes/github');
 const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 const repoAddress = 'https://github.com/apache/incubator-opendal';
 
+const baseUrl = process.env.BASE_URL ? process.env.BASE_URL : '/';
+const showAnnouncementBar = process.env.SHOW_ANNOUNCEMENT_BAR ? process.env.SHOW_ANNOUNCEMENT_BAR : false;
```

Comment:
Please use `OPENDAL_WEBSITE_SHOW_ANNOUNCEMENT_BAR`. By the way, the naming seems confusing. How about use `OPENDAL_WEBSITE_NOT_LATEST`?

Reply:
SGTM